bokeh/models/formatters.py

```diff
@@ -282,26 +283,7 @@ def from_coffeescript(cls, code, args={}):
 
     """)
 
-DEFAULT_DATETIME_FORMATS = lambda : {
-    'microseconds': ['%fus'],
-    'milliseconds': ['%3Nms', '%S.%3Ns'],
-    'seconds':      ['%Ss'],
-    'minsec':       [':%M:%S'],
-    'minutes':      [':%M', '%Mm'],
-    'hourmin':      ['%H:%M'],
-    'hours':        ['%Hh', '%H:%M'],
-    'days':         ['%m/%d', '%a%d'],
-    'months':       ['%m/%Y', '%b%y'],
-    'years':        ['%Y'],
-}
```

Comment:
I think we have to leave this for now, with a deprecation. Otherwise people that use it currently will break immediately, rendering the other deprecation of `.formats` useless.